_posts/08-01-01-Templating.md

```diff
@@ -1,13 +1,12 @@
 ---
 isChild: false
-title: Templating
+title: Templejti
```

Comment:
Da, moglo bi možda *Korišćenje template-a* ili *Rad sa template-ima*, tako nešto.